src/com/paynemiller/ch4/TaxCalculator.java

```diff
@@ -0,0 +1,50 @@
+package com.paynemiller.ch4;
+
+import java.util.Scanner;
+
+//exercise 14
+public class TaxCalculator {
+    Scanner keyboard = new Scanner(System.in);
+    double orderAmount;
+    String stateInitials;
+    double subtotal;
+    double tax;
+    double total;
+//primitive data types / abstract data types
```

Comment:
Let's remove these comments. They send more like notes than comments about the code!

Reply:
Removed as requested.